fe/src/main/java/org/apache/doris/load/Load.java

```diff
@@ -651,11 +651,15 @@ public static void checkAndCreateSource(Database db, DataDescription dataDescrip
             for (Column column : tableSchema) {
                 nameToTableColumn.put(column.getName(), column);
             }
-
-            // source columns
             List<String> columnNames = Lists.newArrayList();
-            List<String> assignColumnNames = dataDescription.getColumnNames();
-            if (assignColumnNames == null) {
+            List<String> assignColumnNames = Lists.newArrayList();
+            if (dataDescription.getColumnNames() != null) {
+                assignColumnNames.addAll(dataDescription.getColumnNames());
+            }
+            if (dataDescription.getColumnsFromPath() != null) {
```

Comment:
If user specify columnFromPath without columnList,
I think we should return user's error message in LoadStatement analyze function.
For here, we should check dataDescription.getColumnsFromPath() in `if (dataDescription.getColumnNames() != null)` block